packages/camera/camera/test/camera_test.dart

```diff
@@ -1268,7 +1268,7 @@ class MockCameraPlatform extends Mock
   Future<int> createCamera(
     CameraDescription description,
     ResolutionPreset? resolutionPreset, {
-    bool enableAudio = true,
+    bool enableAudio = false,
```

Comment:
This change, is it to fix another issue?

Reply:
There was a warning when running `flutter analyze` on the project. This is a mock implementation of the the `CameraPlatform` class which got updated during the null-safety migration. During the migration we missed updating this value. 

Since `flutter analyze` was complaining I decided to update it with this PR.